hunt/config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  get 'animals/index'
+  get 'animals/show'
+  get 'animals/new'
+  get 'animals/edit'
```

Comment:
You can get rid of these lines as well as the commented out `#get 'animals/index'`. Because you added `resources :animals` we get all of the following routes:
```
animals     GET    /animals(.:format)                 animals#index
            POST   /animals(.:format)                 animals#create
new_animal  GET    /animals/new(.:format)             animals#new
edit_animal GET    /animals/:id/edit(.:format)        animals#edit
animal      GET    /animals/:id(.:format)             animals#show
            PATCH  /animals/:id(.:format)             animals#update
            PUT    /animals/:id(.:format)             animals#update
            DELETE /animals/:id(.:format)             animals#destroy
```

To see all of the routes you have defined in this file, run `rake routes`